### PR TITLE
Template commit

### PR DESCRIPTION
**Pull request by TadasRup**
**Named sections**

### CODE REVIEW

**Review by AndriusCa — CHANGES_REQUESTED** [review]
change footer comment where`s written Header

**Review by MarekAkram — DISMISSED** [review]

**Comment by TadasRup** [comment]
I fixed few comments in index.html

**Review by Vilteee — DISMISSED** [review]

**Comment by Vilteee** [comment]
all good

**Review by MarekAkram — DISMISSED** [review]

**Review by Vilteee — CHANGES_REQUESTED** [review]
Need to add testimonials start, end comments to home.js

**Comment by TadasRup** [comment]
Changed comments

**Review by GiedreRag — DISMISSED** [review]

File: js/pages/home.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+/* METRICS: start */
+/* METRICS: end */
+
+/* VIDEO: start */
+/* VIDEO: end */
+
+/* SOCIALS: start */
+/* SOCIALS: end */
+
```
Comment (M1r4cl38) [inline js/pages/home.js:18]: testimonials: start
testimonials: end

**Review by M1r4cl38 — CHANGES_REQUESTED** [review]
missed testimonials section in .js

**Review by dittamaz — APPROVED** [review]

File: js/pages/home.js (hunk 1 above)
Comment (TadasRup) [reply js/pages/home.js:18]: Fixed

**Comment by TadasRup** [comment]
Added:
testimonials: start
testimonials: end
In home.js

**Review by M1r4cl38 — APPROVED** [review]
well done

**Review by AndriusCa — APPROVED** [review]

**Review by Vilteee — APPROVED** [review]

**Pull request merged**